blog-mysql-seed/mysql/mysql.star

```diff
@@ -0,0 +1,52 @@
+MYSQL_IMAGE = "mysql:8.0.32"
+
+def create_database(plan, database_name, database_user, database_password, seed_script_artifacts = []):
```

Comment:
And same here, not sure where this is called

Reply:
This will be called from another package